client/src/components/HistoryViewer/HistoryViewerVersionList.js

```diff
@@ -42,28 +49,23 @@ class HistoryViewerVersionList extends PureComponent {
    * @param {Object} version
    * @returns {boolean}
    */
-  isVersionActive(version) {
-    const { currentVersion, compare, compare: { versionFrom, versionTo } } = this.props;
-
+  const isVersionActive = (version) => {
+    const { versionFrom, versionTo } = compare || {};
     const isCurrent = currentVersion && currentVersion.version === version.version;
     const isCompareFrom = versionFrom && versionFrom.version === version.version;
     const isCompareTo = versionTo && versionTo.version === version.version;
-
-    return (!compare && isCurrent) || isCompareFrom || isCompareTo;
-  }
+    return ((!compare && isCurrent) || isCompareFrom || isCompareTo) || false;
```

Comment:
Why was this condition changed?

Reply:
compare (compareType) can be false. 
HistoryViewerVersionList-test.js adds a lot of tests include some that test it with compare = false. Previously that state was not tested.

Reply:
> compare (compareType)

What is `compareType`?

> HistoryViewerVersionList-test.js adds a lot of tests include some that test it with compare = false. Previously that state was not tested.

That still doesn't explain why it was changed - and why `|| false` is the appropriate part to add. Won't the `|| false` part only happen if `((!compare && isCurrent) || isCompareFrom || isCompareTo)` has already evaluated falsy, making it redundant?

Reply:
Can you please add a comment to the code to explain the use of `|| false` here for when we look at this in the future? Something like this will do:
`// Convert falsy values like undefined to a proper boolean false`

Reply:
Added